algebird-core/src/main/scala/com/twitter/algebird/Ring.scala

```diff
@@ -139,6 +140,7 @@ object Ring extends GeneratedRingImplicits with ProductRings {
   implicit val jshortRing: Ring[JShort] = JShortRing
   implicit val longRing: Ring[Long] = LongRing
   implicit val bigIntRing: Ring[BigInt] = BigIntRing
+  implicit val bigDecimalRing: Ring[BigDecimal] = BigDecimalRing
```

Comment:
Well. I didn't go that far. My initial motivation was that eventually I no longer need to provide my own ```Semigroup[BigDecimal]```; as ```Ring[BigInt]``` was here and is used by ``Semigroup[BigInt]```, ```Monoid[BigInt]```, ```Group[BigInt]```, I just xerox'd the pattern.